examples/translate_ui.R

```diff
@@ -0,0 +1,40 @@
+#' This minimal example demonstrates translating ui elements using shiny.18n
```

Comment:
This is not an accurate description - this example is more about adding minimal style to the language dropdown.